cluster/juju/bundles/README.md

```diff
@@ -74,7 +74,7 @@ You may also `juju ssh kubernetes-master/0` and call kubectl from that
 machine.
 
 See the
-[kubectl documentation](https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/kubectl.md)
+[kubectl documentation](https://github.com/kubernetes/kubernetes/blob/master/docs/kubectl.md)
```

Comment:
I think you get the point - docs links can go through docs.k8s.io, releases.k8s.io for release links and tagged code, code.k8s.io for HEAD code, etc.